code/modules/reagents/reagents/reagents_toxin.dm

```diff
@@ -479,6 +479,51 @@
 		O.visible_message("<span class='warning'>\The [O] melts.</span>")
 		qdel(O)
 
+/datum/reagent/mutagen/metastable
+	name = "Metastable Mutagen"
+	id = METASTABLE_MUTAGEN
+	description = "Causes controlled mutations in plants and tables."
+
+/datum/reagent/mutagen/metastable/New()
+	..()
+	name = pick("Metatable Mutagen", "Metastable Mutagen") //it's funny to me
+
+/datum/reagent/mutagen/metastable/reaction_obj(var/obj/O, var/volume)
+	if(..())
+		return 1
+
+	if(!(O.dissolvable() == PACID))
+		return
+	var/list/tabletypes = list(/obj/structure/table,
+								/obj/structure/table/woodentable,
+								/obj/structure/table/woodentable/poker,
+								/obj/structure/table/glass,
+								/obj/structure/table/glass/plasma,
+								/obj/structure/table/plastic,
+								/obj/structure/table/reinforced,
+								/obj/structure/table/reinforced/clockwork
```

Comment:
No. Make a separate reagent for this meme.

Reply:
It's not a meme, it's a legit implementation for a chem.
Untable is the botanic equivalent for Unstable and melts tables. 
It made sense that the stabilized variant of it would morph tables instead, cause our plant mutagens have an effect outside of botany, this fit the bill.